compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -5031,6 +5055,22 @@ private boolean isReadOnlyAndObjectIntersection(BIntersectionType referredType)
         return false;
     }
 
+    private boolean withinNode(LineRange srcRange, LineRange targetRange) {
```

Comment:
According to the implementation, the duty of this function is to compare ranges. IMO, the name should be `withinRange`.